test/resources/Conferences.spec.js

```diff
@@ -84,6 +84,21 @@ describe('Calls Resource', function() {
                 .then(responseFn);
             })
         });
+
+        it('Sends the correct request [with empty resource instance]', function() {
```

Comment:
I'm noticing these specs are also structured similarly. If the resources are very similar and you're just testing a list of commands, maybe you can abstract it.

Reply:
@pstoica I will open a new PR with spec updates, including test coverage increase

Reply:
thanks!